app/utils/fetch-selection-summary.js

```diff
@@ -4,13 +4,32 @@ const { SupportServiceHost } = Environment;
 
 const SELECTION_SUMMARY_API_URL = (survey = 'decennial', summaryVars = 'pop1,hunits', geoid = '0') => `${SupportServiceHost}/summary/${survey}/${summaryVars}/${geoid}`;
 
+const maxGeoidLength = 500;
 
 export default async function fetchSelectionSummary(survey, summaryVars, geoid) {
-  if(geoid.length) {
-    let selectionResponse = null;
-    selectionResponse = await fetch(SELECTION_SUMMARY_API_URL(survey, summaryVars, geoid));
-    selectionResponse = await selectionResponse.json();
-    return selectionResponse;
+  try {
+    let selectionResponse = {totals: { hunits: 0, pop1: 0 }};
+    // to ensure we don't receive errors from the URLs being too long, we limit the selections to 500 geoids
+    // The limit can be increased when indices are added to the geoids in the pff db
+    if (geoid.length > maxGeoidLength) {
+      selectionResponse = null;
+
+      /* eslint-disable-next-line no-console */
+      console.log(`GEOID_SELECTION_ERROR: geoid selection of ${geoid.length} is greater than than permitted maximum of ${maxGeoidLength}`);
```

Comment:
I thought it might be a good idea to have these errors accessible in the server logs. WHat do y'all think?

Reply:
Server logs would be nice but this is the front end so this won't show up there. You can remove this.

Reply:
That's right! We gotta throw this thing onto  Sentry.